source/ConfigSetup.h

```diff
@@ -43,6 +43,10 @@ EMP_BUILD_CONFIG(SymConfigBase,
     VALUE(FREE_LIVING_SYMS, bool, 0, "Are symbionts able to live outside of the host?"),
     VALUE(MOVE_FREE_SYMS, bool, 0, "Should the free living phage move around in the world? 0 for no, 1 for yes"),
     
+    GROUP(PGG,"Public Goods Game Settings"),
```

Comment:
This should go after the file_path and file_name since otherwise they will be lumped into the PGG group